src/components/Alert/Alert.module.scss

```diff
@@ -71,15 +71,19 @@
     }
   }
 
+  .alert-title {
+    &:not(:last-child) {
+      margin-bottom: var(--alert-title-margin);
+    }
+  }
+
   .alert-message {
     &:not(:last-child) {
       margin-right: var(--alert-message-margin);
     }
-  }
-}
 
-.alert-title {
-  &:not(:last-child) {
-    margin-bottom: var(--alert-title-margin);
+    > p {
```

Comment:
Since .alert-message can only have one `<p>` as a direct descendent, I removed the bottom margin from it. However, if this `<p>` has any `<p>` descendents, they should still have their bottom margins applied, to maintain regular spacing.